docs/source/ref/models/search.rst

```diff
@@ -0,0 +1,626 @@
+given list.
+
+This expression uses the :doc:`in operator <atlas:atlas-search/in>` to match
+documents whose field contains a value from the provided array.
+
+.. code-block:: pycon
+
+    >>> from django_mongodb_backend.expressions import SearchIn
+    >>> Article.objects.annotate(score=SearchIn(path="number", value=[1, 2]))
+    <QuerySet [
+        <Article: title: Introduction to Atlas Search (number=1)>,
+        <Article: title: Boosting Relevance Scores (number=2)>
+    ]>
+
+The ``path`` argument can be the name of a field (as a string) or a
+:class:`~django.db.models.F`. The ``value`` must be a list
+of values or a :class:`~django.db.models.Value`.
+
+An optional ``score`` :class:`SearchScoreOption` argument to tune the
+relevance score.
+
+``SearchPhrase``
+----------------
+
+Atlas Search expression that matches a phrase in the specified field.
+
+This expression uses the :doc:`phrase operator <atlas:atlas-search/phrase>` to
+find exact or near-exact sequences of terms. It supports optional slop
+(term distance) and synonym mappings defined in the Atlas Search index.
+
+.. code-block:: pycon
+
+    >>> from django_mongodb_backend.expressions import SearchPhrase
+    >>> Article.objects.annotate(
+    ...     score=SearchPhrase(path="body", query="climate change", slop=2)
+    ... )
+    <QuerySet [
+        <Article: title: Understanding Climate Change Models>,
+        <Article: title: The Impact of Rapid Change in Climate Systems>
+    ]>
+
+The ``path`` argument specifies the field to search and can be a string or a
+:class:`~django.db.models.F`. The ``query`` is the phrase to
+match, passed as a string or a list of strings (terms).
+
+Optional arguments:
+
+- ``slop``: The maximum number of terms allowed between phrase terms.
+- ``synonyms``: The name of a synonym mapping defined in your Atlas index.
+- ``score``: A :class:`SearchScoreOption` to tune the relevance score.
+
+``SearchQueryString``
+---------------------
+
+Atlas Search expression that matches using a Lucene-style query string.
+
+This expression uses the
+:doc:`queryString operator <atlas:atlas-search/queryString>` to parse and
+execute full-text queries written in a simplified Lucene syntax. It supports
+features like boolean operators, wildcards, and field-specific terms.
+
+.. code-block:: pycon
+
+    >>> from django_mongodb_backend.expressions import SearchQueryString
+    >>> Article.objects.annotate(
+    ...     score=SearchQueryString(path="body", query="django AND (search OR query)")
+    ... )
+    <QuerySet [
+        <Article: title: Building Search Features with Django>,
+        <Article: title: Advanced Query Techniques in Django ORM>
+    ]>
+
+The ``path`` argument can be a string or a
+:class:`~django.db.models.F` representing the field to query.
+The ``query`` argument is a Lucene-style query string.
+
+An optional ``score`` :class:`SearchScoreOption` argument to tune the
+relevance score.
```

Comment:
This isn't a standalone sentence. Suggestion:  "The optional score argument is a SearchScoreOption that tunes the relevance score."